packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -717,7 +724,14 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
       );
     }
 
-    return _configuration.buildViewportChrome(context, result, widget.axisDirection);
+    return _configuration.buildViewportDecoration(
+      context,
+      result,
+      ScrollableDetails(
+        direction: widget.axisDirection,
+        controller: widget.controller ?? ScrollController(),
```

Comment:
This scroll controller needs to be properly owned by the State: You don't want to create a new one on every build and at the end of its useful life it needs to be disposed.